packages/ui-color-picker/src/ColorPicker/index.tsx

```diff
@@ -0,0 +1,556 @@
+              }
+            />
+          </div>
+        )}
+        {this.props?.colorMixerSettings?.colorContrast && (
+          <div css={this.props.styles?.popoverContentBlock}>
+            <ColorContrast
+              firstColor={
+                this.props.colorMixerSettings.colorContrast.firstColor
+              }
+              secondColor={`#${this.stripAlphaIfNeeded(this.state.mixedColor)}`}
+              label={this.props.colorMixerSettings.colorContrast.label}
+              successLabel={
+                this.props.colorMixerSettings.colorContrast.successLabel
+              }
+              failureLabel={
+                this.props.colorMixerSettings.colorContrast.failureLabel
+              }
+              normalTextLabel={
+                this.props.colorMixerSettings.colorContrast.normalTextLabel
+              }
+              largeTextLabel={
+                this.props.colorMixerSettings.colorContrast.largeTextLabel
+              }
+              graphicsTextLabel={
+                this.props.colorMixerSettings.colorContrast.graphicsTextLabel
+              }
+              firstColorLabel={
+                this.props.colorMixerSettings.colorContrast.firstColorLabel
+              }
+              secondColorLabel={
+                this.props.colorMixerSettings.colorContrast.secondColorLabel
+              }
+            />
+          </div>
+        )}
+      </div>
+      <div css={this.props.styles?.popoverFooter}>
+        <Button
+          onClick={() => {
+            this.setState({
+              openColorPicker: false,
+              hexCode: `${this.stripAlphaIfNeeded(this.state.mixedColor)}`
+            })
+          }}
+          color="primary"
+          margin="xx-small"
+        >
+          Add
+        </Button>
+        <Button
+          onClick={() =>
+            this.setState({
+              openColorPicker: false,
+              mixedColor: this.state.hexCode
+            })
+          }
+          color="secondary"
+          margin="xx-small"
+        >
+          Close
+        </Button>
+      </div>
+    </React.Fragment>
+  )
+  render() {
+    const {
+      checkContrast,
+      colorMixerSettings,
+      disabled,
+      elementRef,
+      isRequired,
+      label,
+      onChange,
+      placeholderText,
+      popverButtonScreenReaderLabel,
+      renderInvalidColorMessage,
+      renderIsRequiredMessage,
+      renderMessages,
+      tooltip,
+      value,
+      width,
+      withAlpha,
+      ...props
+    } = this.props
+    return (
+      <div css={this.props.styles?.colorPicker} ref={this.handleRef}>
+        <TextInput
+          {...passthroughProps(props)}
+          isRequired={isRequired}
+          disabled={disabled}
+          renderLabel={() => this.renderLabel()}
+          display="inline-block"
+          width={width}
+          placeholder={placeholderText}
+          themeOverride={{ padding: '0 0.75rem 0 0' }}
```

Comment:
Actually, broken padding did kinda what I needed, so if I just override it to empty string (`themeOverride={{ padding: '' }}`), it will set the padding to 0, which is ok